configure.ac

```diff
@@ -45,7 +45,7 @@ AC_ARG_WITH(
     [ CRM_DAEMON_USER="hacluster" ],
   )
 
-AM_INIT_AUTOMAKE([no-define])
+AM_INIT_AUTOMAKE([foreign])
```

Comment:
This modification allows `automake` and `autoconf` to ignore the fact that there is no `README` file with no extensions. The `README` file can be implemented via distribution package specific means.

Typically, I would not suggest to change this if it was set to gnu - though having been set to no-define makes this perfectly acceptable IMHO.